I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml

```diff
@@ -371,17 +378,19 @@
 ]]></artwork>
   </figure>
   
-  <t>
-   According to <xref target="G.807"/> and <xref target="G.872"/>, an
-   OMS Media Channel Group (MCG) represents a topological construct
-   between two WDM-TE-nodes.
-  </t>
+
   
   <t>
-   Specifically, it originates at the ROADM in the source WDM-TE-node
-   and terminates at the ROADM in the destination WDM-TE-node including
-   the Booster Amplifier (BA) and the Pre-Amplifier (PA) in the
-   WDM-TE-nodes as well as the In-Line Amplifiers (ILAs) between the
+   in contrast, <xref target="G.807"/> describes an OMS MCG as "The 
+   topological relationship between the media port on a filter or coupler
+   where a set of media channels are aggregated and the media port on a 
+   filter or coupler where one or more media channel is added to or removed 
+   from that aggregate."
+
+   An OMS MCG originates at the ROADM in the source 
+   WDM-TE-node and terminates at the ROADM in the destination WDM-TE-node 
+   traversing the Booster Amplifier (BA) and the Pre-Amplifier (PA) in
+   the WDM-TE-nodes as well as the In-Line Amplifiers (ILAs) between the
    two WDM-TE-nodes.
```

Comment:
I do not see WDM TE-node in this section. Figure 2 just shows a WDM node.

IMHO, it is better to use the term WDM node in this paragraph and to reference Figure 2.

Reply:
```suggestion
   An OMS MCG originates at the ROADM in the source 
   WDM-node and terminates at the ROADM in the destination WDM-node 
   traversing the Booster Amplifier (BA) and the Pre-Amplifier (PA) in
   the WDM-nodes as well as the In-Line Amplifiers (ILAs) between the
   two WDM-nodes.
```